### PR TITLE
Yank - Rich link and meta in Crit CSS

### PR DESCRIPTION
**Pull request by gtrufitt**
## What does this change?
- Splits rich links css into head/async and loads the 'default' rich link css in the head
- Moves some of the meta layout css into the head
- Deletes some un-needed stuff

I know that I am adding stuff to head css here, so look for a PR in the future that splits https://github.com/guardian/frontend/blob/master/static/src/stylesheets/module/content/_article-immersive.scss into critical/non-critical css because there's no way that entire file needs to be in the head :D
## What is the value of this and can you measure success?

This fixes first paint update of rich links.
## Screenshots

| Before | After |
| --- | --- |
| critical-css-before | critical-css-after |
## Request for comment

@guardian/dotcom-platform

### CODE REVIEW

File: static/src/stylesheets/module/onward/_rich-links-enhanced.scss [hunk 1]
```diff
@@ -51,22 +38,16 @@
...
 
-.rich-link__title {
-    padding: 0;
-}
-
-.rich-link .rich-link__header {
-    padding: $gs-baseline / 3 $gs-gutter / 4 $gs-baseline;
-    box-sizing: border-box;
-    min-height: $gs-baseline * 3;
+.rich-link__byline {
```
Comment (dominickendrick) [inline static/src/stylesheets/module/onward/_rich-links-enhanced.scss]: Is this a duplication of  [L57](https://github.com/guardian/frontend/pull/14118/files#diff-bbc8a915c2b5cac7c83501927eb823dcL57) ?

Comment (gtrufitt) [inline static/src/stylesheets/module/onward/_rich-links-enhanced.scss]: So it is!

**Comment by johnduffell** [comment]
the right hand side is clearly a static image... 🎉

**Comment by gtrufitt** [comment]
@johnduffell I know right, if it wasn't for the 'connected to browsersync' message you'd never believe I was continually refreshing.

**Comment by gtrufitt** [comment]
Tbf, I think I cheated as I had ad blocking on :/

**Comment by dominickendrick** [comment]
👍

**Comment by paperboyo** [comment]
💎

**Pull request merged**